util/dvsim/SimCfg.py

```diff
@@ -465,14 +473,19 @@ def gen_results_sub(items, results, fail_msgs):
                 map_full_testplan=self.map_full_testplan)
             results_str += "\n"
             self.final_total = self.testplan.final_total
```

Comment:
Just pick "passing", "total" and "pass_rate" so that we can add the "name" later (use self.name instead). We can then link the name to the full detailed results.

Reply:
Also change `testplan.final_total` to `testplan.result`. Convert it into a dict instead of a list so that we can look up the information we want here.